src/helpers/constants.js

```diff
@@ -36,6 +36,13 @@ export const constants = {
       description:
         "Streaming services API contains endpoints to stream account, allowance, facilities, and emissions data",
     },
+    {
+      name: "camd-services",
+      url: "https://api.epa.gov/easey/camd-services/swagger-json",
+      title: "CAMD Services",
+      description:
+        "CAMD services API contains an endpoint for facility submission files (I.e., emissions, monitoring plans, QA), apportioned emissions (including MATS), allowance, compliance, and facility data",
```

Comment:
Maybe we should add something to indicate the files at the endpoint are in bulk to sort of lead people who were looking for the FTP?